src/modules/atommodel/particles/RadixParticle.spec.ts

```diff
@@ -32,16 +32,11 @@ describe('RadixParticle', () => {
     {
         const chronoQuark1 = new RadixChronoQuark('a', 1)
         const chronoQuark2 = new RadixChronoQuark('b', 2)
-        const dataQuark = new RadixDataQuark('', {})
 
         const particle = new RadixParticle(chronoQuark1, chronoQuark2, dataQuark)
 
         it(`should return all quarks`, () => {
-            expect(particle.quarks).to.have.lengthOf(3)
-        })
-
-        it(`should filter 1 quark`, () => {
-            expect(particle.getQuarks(RadixDataQuark)).to.have.lengthOf(1)
+            expect(particle.quarks).to.have.lengthOf(2)
```

Comment:
This test doesn't test what it says it does, but since we're getting rid of all quarks and all of this test will be invalidated, I think it's ok.